web/src/message_edit.js

```diff
@@ -1237,7 +1237,7 @@ export function delete_topic(stream_id, topic_name, failures = 0) {
     });
 }
 
-export function handle_narrow_deactivated() {
+export function restore_edit_state_for_msg_list() {
```

Comment:
`restore_edit_state_for_msg_list` => ` restore_edit_state_after_message_view_change` might be more readable.

Reply:
Done